nodejs/awsx/apigateway/api.ts

```diff
@@ -566,7 +595,7 @@ export class API extends pulumi.ComponentResource {
             // when needed.  The Stage allocated below will be the stable stage that always points
             // to the latest deployment of the API.
             variables: {
-                version: swaggerString.apply(s => sha1hash(s)),
+                version: argsHash.apply(s => sha1hash(s)),
```

Comment:
small problem in that this impacts all existing deployments.  @lukehoban do we have some way of preventing that?  i'm not even sure how we would express this as a concept. 

However, since this is in a prerelease package.  Perhaps it's ok to do this, just release in a minor-version to indicate the compat break?

Reply:
I'd suggest that triggering a redeployment of all existing deployments may be beneficial as we recognised this issue due to the fact that one of our APIs wouldn't redeploy (posing a potential security risk). Could potential updates be made to the pulumi preview/up commands to request consent for this redeployment?